From 8f226ccc9d2cbd10d454e131376c35f502a45e3b Mon Sep 17 00:00:00 2001
From: Chase Maupin <Chase.Maupin@ti.com>
Date: Wed, 7 Mar 2012 14:11:42 -0600
Subject: [PATCH] Makefile: allow building with OE

* Use defaults and source values that allow building this
  component with OE.

Upstread-Status: Pending
    * Will be accepted into next release

Signed-off-by: Chase Maupin <Chase.Maupin@ti.com>
---
 Makefile |   21 +++++++++++++++++++++
 1 files changed, 21 insertions(+), 0 deletions(-)

diff --git a/Makefile b/Makefile
index 9035e40..bd94881 100644
--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,26 @@
 # Installs the the Bluetooth firmware files into the root file system

+-include ../../../Rules.make
+
+# If KERNEL_DIR is not set then use the default in Rules.make
+DEST_DIR ?= ${DESTDIR}
+
+PLATFORM ?= "unknown"
+MACHINE_NAME ?= "unknown"
+
+ifeq ($(PLATFORM), am335x-evm)
+    MACHINE_NAME := am335x
+endif
+ifeq ($(PLATFORM), am180x-evm)
+    MACHINE_NAME := am1808
+endif
+ifeq ($(PLATFORM), da850-omapl138-evm)
+    MACHINE_NAME := am1808
+endif
+ifeq ($(PLATFORM), am37x-evm)
+    MACHINE_NAME := omap3evm
+endif
+
 install:
	install -d $(DEST_DIR)${BASE_LIB_DIR}/firmware
	cp -f ./${MACHINE_NAME}/* $(DEST_DIR)${BASE_LIB_DIR}/firmware
--
1.7.0.4
